### ISSUE TITLE
[Feature request] CC module for Hide Jailbreak option

### ISSUE THREAD

**Issue by Void48** [opa334/Dopamine#552]
Would be extremely useful since it would save some time and make it more easy to activate the option

**Comment by luhgarlicbread** [comment]
The vnodebypass tweak has exactly this!

**Comment by klearnstodev** [comment]
Second that.
Since dopamine has the OFFICIAL support to temporarily hide the jailbreak, I have no idea that no one including the opa334 himself does not plan to develop a shortcut to activate this function, just as what vnodebypass has done. It’s really brilliant to activate/deactivate by a CC toggle. Maybe they had less bank apps installed, lol. But sooner or later, most bank apps are going to detect rootless jailbreak, just like the old rootful days.
So I kindly ask opa334 to consider this request.
As for the alternative vnodebypass right now, the ‘USE IT AT YOUR RISK!’ opening sign really scared me, lol.

**Comment by Void48** [comment]
> Second that. Since dopamine has the OFFICIAL support to temporarily hide the jailbreak, I have no idea that no one including the opa334 himself does not plan to develop a shortcut to activate this function, just as what vnodebypass has done. It’s really brilliant to activate/deactivate by a CC toggle. Maybe they had less bank apps installed, lol. But sooner or later, most bank apps are going to detect rootless jailbreak, just like the old rootful days. So I kindly ask opa334 to consider this request. As for the alternative vnodebypass right now, the ‘USE IT AT YOUR RISK!’ opening sign really scared me, lol.

I think the most convenient and ideal way would be to just use RootHide version of Dop2, as it eliminates the need to constantly hide the jailbreak (and still, even this has its own drawbacks)

**Comment by klearnstodev** [comment]
> > Second that. Since dopamine has the OFFICIAL support to temporarily hide the jailbreak, I have no idea that no one including the opa334 himself does not plan to develop a shortcut to activate this function, just as what vnodebypass has done. It’s really brilliant to activate/deactivate by a CC toggle. Maybe they had less bank apps installed, lol. But sooner or later, most bank apps are going to detect rootless jailbreak, just like the old rootful days. So I kindly ask opa334 to consider this request. As for the alternative vnodebypass right now, the ‘USE IT AT YOUR RISK!’ opening sign really scared me, lol.
> 
> I think the most convenient and ideal way would be to just use RootHide version of Dop2, as it eliminates the need to constantly hide the jailbreak (and still, even this has its own drawbacks)

Yes, indeed. But I tried and gave up many times at the thought of having to do too many conversions for tweaks at the risk of the compatibility, lol.

**Comment by ghh-jb** [comment]
Not related to dopamine. Go to r/tweakbounty or to r/jailbreakdevelopers. It is not an issue and it is not connected with dopamine.
